tests/epyccel/modules/base.py

```diff
@@ -42,6 +42,20 @@ def compare_is_not_int(a, b):
         c = True
     return c
 
+@types('int', 'int')
+def compare_is_int_int(a, b):
+    c = False
+    if a is b:
+        c = True
+    return c
+
+@types('int', 'int')
+def compare_isnot_int_int(a, b):
+    c = False
+    if a is not b:
+        c = True
+    return c
+
```

Comment:
Do we have an example where we use `is` to compare objects of different types?

Even if it makes little sense, I think we should implement it: we know a priori that `is` will return `False` in that case.

Reply:
yes i agree, im gonna implement it